docs/usage/Installations.md

```diff
@@ -0,0 +1,459 @@
+    dataRef: ""
+  - name: my-import
+    secretRef: 
+      name: ""
+      namespace: ""
+      key: ""
+  - name: my-import
+    configMapRef: 
+      name: ""
+      namespace: ""
+      key: ""
+```
+
+### Target Imports
+
+Target imports are defined by a unique logical name and a reference to the actual target.
+
+The logical name must be unique within all imports (data and targets) and must match the import name in the blueprint.<br>
+The `target` attribute defines the reference to a real target within the same namespace.
+By default the target name is [contextified](../concepts/Context.md) but 
+a target can also be directly referenced by its name by prefixing the `target` with a `#`.
+
+```yaml
+imports:
+  targets:
+  - name: my-target
+    target: ""
+```
+
+
+### Import Data Mappings
+
+It can happen that imported data is of a different format than the expected schema defined in the blueprint.<br>
+One possible solution is to add an additional blueprint that transforms the data.
+As this approach would result in a big overhead for just transforming some data. 
+It should be possible to easily transform imported data into imports of blueprints.
+
+This transformation can be done in `spec.importDataMappings`.
+`ImportDataMappings` define a map of all imports of a blueprint that can be templated using [spiff](https://github.com/mandelsoft/spiff). <br>
+These mappings make it possible to define imports using multiple imports, use hard coded values or use parts of an import.
+
+All imported values can be used in the templating by their logical internal names.
+All blueprint import data mappings are optional, by default the logical internal name is matched to the blueprint import.
+```yaml
+spec:
+  imports:
+    data:
+    - name: imp1
+    - name: imp2
+  importDataMappings:
+    <blueprintimport1>: hardcoded value
+    <blueprintimport2>: (( imp1.subkey ))
+    <blueprintimport3>: 
+      a: (( imp1 ))
+      b: (( imp2 ))
+```
+
+__Example__
+```yaml
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+imports:
+- name: providers
+  schema:
+    type: array
+    items:
+      type: string
+- name: identifier
+  schema:
+    type: string
+- name: aws-credentials
+  schema:
+    type: object
+    properties:
+      accessKeyID:
+        type: string
+      accessKeySecret:
+        type: string
+```
+
+```yaml
+spec:
+  imports:
+    data:
+    - name: aws-provider #  value: { "type": "aws", "creds": { "accessKeyID": "adfa", "accessKeySec": "1234" } } }
+    - name: gcp-provider-type #  value: "gcp"
+  importDataMappings:
+    identifier: my-controller
+    providers:
+    - (( aws-provider-type.type ))
+    - (( gcp-provider-type ))
+    aws-credentials: 
+      accessKeyID: (( aws-provider-type.creds.accessKeyID ))
+      accessKeySecret: (( aws-provider-type.creds.accessKeySec ))
+```
+
+
+## Exports
+
+Exports define the data that is created by the installation.
+
+Whom can distinguish between two basic types of exports:
```

Comment:
Same as above - I think this 'whom' phrase sounds weird. I suggest a similar rephrasing as mentioned above.

Reply:
Just a short question, since this is the only part that you didn't change - did you skip this one on purpose or did you miss it?

Reply:
Just missed that one